stats/lib/handler.js

```diff
@@ -141,86 +144,14 @@ const notFound = (res) => {
 }
 
 /**
- * @param {import('node:http').IncomingMessage} _req
+ * @param {import('node:http').IncomingMessage} req
  * @param {import('node:http').ServerResponse} res
- * @param {PgPools['api']} client
- * @param {string} minerId
+ * @param {string} SPARK_API_BASE_URL
  */
-const getRetrievableDealsForMiner = async (_req, res, client, minerId) => {
-  /** @type {{rows: {client_id: string; deal_count: number}[]}} */
-  const { rows } = await client.query(`
-    SELECT client_id, COUNT(cid)::INTEGER as deal_count FROM retrievable_deals
-    WHERE miner_id = $1 AND expires_at > now()
-    GROUP BY client_id
-    ORDER BY deal_count DESC, client_id ASC
-    `, [
-    minerId
-  ])
-
-  // Cache the response for 6 hours
-  res.setHeader('cache-control', `max-age=${6 * 3600}`)
-
-  const body = {
-    minerId,
-    dealCount: rows.reduce((sum, row) => sum + row.deal_count, 0),
-    clients:
-      rows.map(
-        // eslint-disable-next-line camelcase
-        ({ client_id, deal_count }) => ({ clientId: client_id, dealCount: deal_count })
-      )
-  }
-
-  json(res, body)
-}
-
-const getRetrievableDealsForClient = async (_req, res, client, clientId) => {
-  /** @type {{rows: {miner_id: string; deal_count: number}[]}} */
-  const { rows } = await client.query(`
-    SELECT miner_id, COUNT(cid)::INTEGER as deal_count FROM retrievable_deals
-    WHERE client_id = $1 AND expires_at > now()
-    GROUP BY miner_id
-    ORDER BY deal_count DESC, miner_id ASC
-    `, [
-    clientId
-  ])
-
+const redirectToSparkApi = async (req, res, SPARK_API_BASE_URL) => {
```

Comment:
Missed this before :)

```suggestion
const redirectToSparkApi = (req, res, SPARK_API_BASE_URL) => {
```